client/components/ManageUsers.js

```diff
@@ -0,0 +1,104 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+class ManageUsers extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      auth: {},
+      users: [],
+    };
+    this.destroy = this.destroy.bind(this);
+    this.makeAdmin = this.makeAdmin.bind(this);
+  }
+
+  componentDidMount() {
+    this.exchangeToken();
+    this.loadUsers();
+  }
+
+  componentDidUpdate() {
```

Comment:
Im not too sure but Im wondering if you need to wrap this in a if statement so it doesnt get caught in an infinite loop?

Reply:
thanks - just saw your approval. I'll go ahead and push and then look into wrapping the componentDidUpdate in an if statement